openlibrary/plugins/books/readlinks.py

```diff
@@ -102,25 +101,17 @@ def get_solr_edition_records(iaids):
 class ReadProcessor:
     def __init__(self, options):
         self.options = options
-        self.set_inlibrary = False
-
-    def get_inlibrary(self):
-        if not self.set_inlibrary:
-            self.set_inlibrary = True
-            self.inlibrary = inlibrary.get_library()
-        return self.inlibrary
-
 
     def get_item_status(self, ekey, iaid, collections, subjects):
         if 'lendinglibrary' in collections:
-            if not 'Lending library' in subjects:
+            if 'Lending library' not in subjects:
                 status = 'restricted'
             else:
                 status = 'lendable'
         elif 'inlibrary' in collections:
-            if not 'In library' in subjects:
+            if 'In library' not in subjects:
                 status = 'restricted'
-            elif not self.get_inlibrary():
+            elif True: # not self.get_inlibrary(): - Deprecated
```

Comment:
I'd probably use `else` here, but this whole method looks like it should be deprecated, so I'd rather get confirmation from @mekarpeles before changing anything else.